src/views/portfolio/projects/ProjectList.vue

```diff
@@ -179,6 +179,22 @@
               return `<a href="${url}">${xssFilters.inHTMLData(value)}</a>`;
             }
           },
+          {
+            title: this.$t('message.tags'),
+            field: "tags",
+            sortable: false,
+            visible: false,
+            formatter(value, row, index) {
+              let tag_string = row.tags.slice(0, 2)?.map(tag => common.formatProjectTagLabel(tag)).join(' ') || '';
```

Comment:
There needs to be an `undefined` / `null` check here, this line is currently failing when a project does not have any tags.

<img width="428" alt="image" src="https://user-images.githubusercontent.com/5693141/232245760-60694009-baf2-45ab-95fd-9967d80a0d8a.png">

Reply:
I am not seeing that error, but added a check anyway.